src/Stateless/ShapeColor.cs

```diff
@@ -0,0 +1,137 @@
+﻿namespace Stateless.Dot
+{
+    enum ShapeColor
```

Comment:
Since the same colors could arguably be used for lines and so-on, is `HtmlColor` a better name for the enclosing class?